std_msgs/Header header
opencv_open/Detection[] detections